app.js

```diff
@@ -142,5 +144,5 @@ window.onload = function() {
         }
     }
 
-    translate()
+    // translate() 
```

Comment:
remove the comment - this will break the production build if it is deployed because the initial translation will not happen on page load

Reply:
fixed and committed